aws_xray_sdk/core/patcher.py

```diff
@@ -80,3 +110,83 @@ def _patch(module_to_patch):
 
     _PATCHED_MODULES.add(module_to_patch)
     log.info('successfully patched module %s', module_to_patch)
+
+
+def _patch_func(parent, func_name, func, modifier=lambda x: x):
+    from aws_xray_sdk.core import xray_recorder
+
+    capture_name = func_name
+    if func_name.startswith('__') and func_name.endswith('__'):
+        capture_name = '{}.{}'.format(parent.__name__, capture_name)
+    setattr(parent, func_name, modifier(xray_recorder.capture(name=capture_name)(func)))
+
+
+def _patch_class(module, cls):
+    for member_name, member in inspect.getmembers(cls, inspect.isclass):
+        if member.__module__ == module.__name__:
+            # Only patch classes of the module, ignore imports
+            _patch_class(module, member)
+
+    for member_name, member in inspect.getmembers(cls, inspect.ismethod):
+        if member.__module__ == module.__name__:
+            # Only patch methods of the class defined in the module, ignore inherited
```

Comment:
are inherited methods patched recursively later?

Reply:
Does this new commit answer that? 😅 https://github.com/yoyowallet/aws-xray-sdk-python/pull/4/commits/07c376289a2a4a93a172e64f5da6f132e814d98f